akka-stream-tests/src/test/scala/akka/stream/impl/fusing/ActorGraphInterpreterSpec.scala

```diff
@@ -13,12 +13,13 @@ import akka.stream.testkit.Utils._
 import akka.stream.testkit.{ StreamSpec, TestPublisher, TestSubscriber }
 import akka.testkit.EventFilter
 
-import scala.concurrent.Await
+import scala.concurrent.{ Await, ExecutionContext, Future }
 import scala.concurrent.duration._
 import org.reactivestreams.{ Publisher, Subscriber, Subscription }
 
 class ActorGraphInterpreterSpec extends StreamSpec {
   implicit val materializer = ActorMaterializer()
+  implicit def ec: ExecutionContext = materializer.executionContext
```

Comment:
Again, these are internal tests, your stage does not exercise anything new that relates to the interpreter. Please move these to the unit tests of your stage and keep this file as it is. In general, you don't need to touch tests under `impl.*` unless you have modified the interpreter or the layout (which is not true for this PR).